js/bench/utils.ts

```diff
@@ -0,0 +1,149 @@
+
+
+import { performance } from "perf_hooks";
+import { access } from 'node:fs/promises';
+import earcut from 'earcut';
+import { classifyRings } from '@maplibre/maplibre-gl-style-spec';
+import { GeometryType } from "../src/data/GeometryType";
+
+export const toKb = (bytes: number) => {
+  return (bytes / 1024).toFixed(2);
+}
+
+export const parseArgs = async (args: string[]): Promise<{ tilePath: string; iterations: number; }> => {
+    const tilePath = args[0];
+    if (!tilePath || !((tilePath.endsWith(".mlt")) || (tilePath.endsWith(".mvt")))) {
+      console.error("Please provide a path to an .mlt or .mvt file");
+      process.exit(1);
+    }
+    await access(tilePath);
+    const iterations = args[1] ? parseInt(args[1]) : 500;
+    if (Number.isNaN(iterations) || iterations < 1) {
+      throw new Error("Please provide a valid number of iterations");
+    }
+    return { tilePath, iterations };
+}
+
+function round(value, precision) {
+  const multiplier = Math.pow(10, precision || 0);
+  return Math.round(value * multiplier) / multiplier;
+}
+
+function msToSec(ms) {
+  return ms / 1000;
+}
+
+function logger(message: string, ops: number, elapsed: number) {
+    const opsPerSec = Math.round(ops/msToSec(elapsed));
+    const msPerOp = round(elapsed/ops,1);
+    console.log('-', opsPerSec, 'ops/s |', msPerOp, 'ms/op', 'for', message, '(runs:', ops,')');
```

Comment:
Not a functional difference but this adds an extra space before closing the parentheses.

```suggestion
    console.log(`- ${opsPerSec} ops/s | ${msPerOp} ms/op for ${message} (runs: ${ops})`);
```